packages/teleterm/src/ui/Tabs/TabItem.tsx

```diff
@@ -43,7 +43,9 @@ export default function TabItem(props: Props) {
       style={{ ...style, opacity }}
     >
       <StyledTabButton>
-        <Text mx="auto">{name}</Text>
+        <Text css={{ direction: 'rtl' }} mx="auto">
```

Comment:
Experimental idea to show `...files/holidays` instead of `/Users/me/fil...` when there is no space, but it will work this way for all kinds of tabs, so I'm not sure if we want it